src/main/java/com/tictactoe/Options.java

```diff
@@ -0,0 +1,53 @@
+package com.tictactoe;
+
+public class Options {
+    private UserInput receiver;
+    private UserInputValidator validator = new UserInputValidator();
+    private String input;
+    public SelectedOptions selectedOptions = new SelectedOptions();
+
+
+    public Options (UserInput receiver) {
+        this.receiver = receiver;
+    }
+
+    public  SelectedOptions collectGamePreference(){
+        boolean allOptionsSelected = false;
+            while(!allOptionsSelected) {
+                System.out.println("Player 1 - Pick your symbol between [X] or [O]: ");
+                this.askForPlayerOneSymbol();
+                System.out.println("Player 2 - Pick your symbol between [X] or [O]: ");
```

Comment:
I'm confused -- if player one picks X can player two pick X as well? If there are two players and only two options, why does player two need to pick a symbol? Seems redundant.